crates/astria-sequencer/src/config.rs

```diff
@@ -0,0 +1,23 @@
+use clap::Parser;
+use serde::{
+    Deserialize,
+    Serialize,
+};
+
+const DEFAULT_ABCI_LISTEN_ADDR: &str = "127.0.0.1:26658";
+
+#[derive(Debug, Deserialize, Parser, Serialize)]
+pub struct Config {
+    /// The endpoint on which Sequencer will listen for ABCI requests
+    #[arg(long, default_value_t = String::from(DEFAULT_ABCI_LISTEN_ADDR))]
+    pub(crate) listen_addr: String,
+    /// The path to the json encoded genesis file with a list of accounts.
+    #[arg(long)]
+    pub(crate) genesis_file: String,
```

Comment:
nice, eventually we need to merge this with the tendermint genesis file, as that's passed in via `InitChain`

Reply:
yeah, I wasn't sure about that. I just blindly followed what was in the tests and figured it would be fine.

Note that I haven't updated the README for this. Can you give this a spin and do that?

Reply:
yep testing it now, will have to make a "test genesis" file for this